packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/shippingMethod.js

```diff
@@ -39,82 +38,78 @@ const ShippingMethod = props => {
         displayState,
         handleCancelUpdate,
         handleSubmit,
-        isLoadingShippingMethods,
-        isLoadingSelectedShippingMethod,
+        isLoading,
         isUpdateMode,
         selectedShippingMethod,
+        setUpdateFormApi,
         shippingMethods,
         showUpdateMode
     } = talonProps;
 
-    const isLoading =
-        isLoadingSelectedShippingMethod || isLoadingShippingMethods;
-
     const classes = mergeClasses(defaultClasses, props.classes);
 
     let contents;
-    switch (displayState) {
-        case displayStates.EDITING:
-            {
-                const continueDisabled =
-                    pageIsUpdating || !shippingMethods.length;
+    if (displayState === displayStates.EDITING) {
+        const lowestCostShippingMethod = {
+            shipping_method: serializeShippingMethod(shippingMethods[0])
```

Comment:
Do we need to serialize the shipping method here? I presume the shipping method in `shippingMethods` is already serealized when returned from `useShippingMethod`.

https://github.com/magento/pwa-studio/pull/2382/files#diff-e8b160b5283e3e60da4aee000580e0bcR145

Reply:
💯 great catch!

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/2382/commits/8053ee17fcc2678896b6d6ecfd6c7678392a5a60.